nss-day-cohort-46/nss-day-cohort-46.github.io
# Mp adding personal info

## Link to Ticket
N/A

TITLE [Link]()

## Description of Proposed Changes
- Added my card and personal info
- I didn't want to change the order too much, but I rearranged a few of the cards so our images complemented those around it. (It's okay if this change is not accepted but I think it looks better for everyone). 

## Steps to Test
- Checkout to my branch
- Make sure that my info is on a card.
- Check how each card complements those beside it.
- See if you like the slight change.

```sh
git fetch --all
git checkout mp-addingPersonalInfo

```


## Impacted Areas in Application

List general components of the application that this PR will affect:
- Only impacts the order of objects (cards) in the cohort array and the corresponding id.
* NONE OF THE CARDS WERE ALTERED. Only the order of a few whose image clashed with one beside it. (Two black and white or two with brick beside each other etc.)

## Mentions @username

Tag users that need to review this code
